src/contracts/libraries/GPv2Order.sol

```diff
@@ -71,7 +71,7 @@ library GPv2Order {
     ///
     /// @param order The order to compute the EIP-712 struct hash for.
     /// @return orderDigest The 32 byte EIP-712 struct hash.
-    function hash(Data memory order)
+    function structHash(Data memory order)
```

Comment:
nanonit: I kind of prefer the old name. When I read structHash I somehow think more about the type hash but really here we have a type + value hash.